path_role.go

```diff
@@ -236,6 +283,15 @@ func (b *jwtAuthBackend) pathRoleCreateUpdate(ctx context.Context, req *logical.
 		role = new(jwtRole)
 	}
 
+	roleType := data.Get("role_type").(string)
+	if roleType == "" {
+		roleType = "jwt"
```

Comment:
I think this should be the other way around. The vast majority of users are likely to want OIDC behavior (and I think we'll be aliasing this plugin as `oidc` within Vault) so if not specified my suggestion is to default to OIDC.

Reply:
agreed

Reply:
I agree re: the user preference, though the reason for the current approach is backwards compatibility. Current users aren't specifying a role type during configuration so they'll need to add `role_type=jwt` to any existing configuration scripts. If that is OK as an upgrade requirement, I have no issues changing it.

Reply:
Thinking about this more, I'm fine with the proposed change.